subxt/src/blocks/blocks_client.rs

```diff
@@ -48,6 +48,36 @@ where
     /// runtime upgrade. You can attempt to retrieve older blocks,
     /// but may run into errors attempting to work with them.
     pub fn at(
+        &self,
+        block_hash: T::Hash,
+    ) -> impl Future<Output = Result<Block<T, Client>, Error>> + Send + 'static {
+        self.at_optional_block_hash(Some(block_hash))
+    }
+
+    /// Obtain block details given the provided block hash, or the latest block if `None` is
+    /// provided.
+    ///
+    /// # Warning
+    ///
+    /// This call only supports blocks produced since the most recent
```

Comment:
We can remove this warning from the documentation since we are always operating with the latest block.
And therefore, users have no way to utilize an older and incompatible block here.